src/components/Steps/Step1Content.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { useHistory } from 'react-router'
+import { IStore } from '../../interfaces/Store'
+import { CLIENT_TYPOLOGY } from '../../enums/journeys'
+import IndividualCompanyStep1Content from '../Journeys/IndividualCompany/Step1/Step1Content'
+import { ROUTES } from '../../enums/routes'
+
+const Step1Content = () => {
+  const { typologie_client } = useSelector(
+    (state: IStore) => state.folderReducer.folder
+  )
+  const { INDIVIDUAL_COMPANY } = CLIENT_TYPOLOGY
+  const history = useHistory()
+
+  switch (typologie_client) {
+    case INDIVIDUAL_COMPANY:
+      return <IndividualCompanyStep1Content />
+    default:
+      history.push(ROUTES.notFound)
+      return <></>
```

Comment:
Remove `history.push(ROUTES.notFound)`
```suggestion
      return <Redirect to={ROUTES.notFound} />
```
Try not to mix js with jsx

Reply:
Done.